### PR TITLE
BALL!

### PR DESCRIPTION
**Pull request by KillianKirilenko**
Tennis balls, that is. Port of [CitRP#612](https://github.com/Citadel-Station-13/Citadel-Station-13-RP/pull/612), minus the squeaky rainbow and donator variants.

Classic tennis balls can be acquired from sweatMAX vendors for a token sum; all variants can be selected in loadout or found as 'clean'/low probability maint items.

### CODE REVIEW

**Comment by Sebbe9123** [comment]
UYES

**Comment by lbnesquik** [comment]
Ball

**Comment by KillianKirilenko** [comment]
Not sure why there's still a queued check on this one; it even links to the successful/passed checks in the details.

**Comment by KasparoVy** [comment]
Szarik!

**Comment by Little-119** [comment]
Balls? owo

File: vorestation.dme [hunk 1]
```diff
@@ -1064,6 +1064,7 @@
 #include "code\game\objects\effects\temporary_visuals\projectiles\tracer.dm"
 #include "code\game\objects\items\antag_spawners.dm"
 #include "code\game\objects\items\apc_frame.dm"
+#include "code\game\objects\items\balls.dm"
```
Comment (Novacat) [inline vorestation.dme:1067]: This should probably have _vr appended at the end just so we know what files are vorestation-specific and which are Polaris files. Icon files should get the same treatment.

Comment (KillianKirilenko) [reply vorestation.dme]: Done.

File: code/game/objects/random/maintenance.dm [hunk 2]
```diff
@@ -107,6 +107,13 @@ something, make sure it's not in one of the other lists.*/
 				prob(2);/obj/item/device/camera,
 				prob(3);/obj/item/device/pda,
 				prob(3);/obj/item/device/radio/headset,
+				prob(2);/obj/item/toy/tennis,
+				prob(2);/obj/item/toy/tennis/red,
+				prob(2);/obj/item/toy/tennis/yellow,
+				prob(2);/obj/item/toy/tennis/green,
+				prob(2);/obj/item/toy/tennis/cyan,
+				prob(2);/obj/item/toy/tennis/blue,
+				prob(2);/obj/item/toy/tennis/purple,
```
Comment (Novacat) [inline code/game/objects/random/maintenance.dm:116]: These touch a polaris file, so this should probably be marked somehow (or moved to a VR file)

Comment (KillianKirilenko) [reply code/game/objects/random/maintenance.dm:116]: I think I was on autopilot at that point. I'll add comment brackets for now and maybe split them off later.

The pizzavoucher needed commenting too anyway.

**Pull request merged**